lib/Ebook.php

```diff
@@ -1561,6 +1592,22 @@ public function Validate(): void{
 			$error->Add(new Exceptions\EbookIndexableTextRequiredException());
 		}
 
+		if(isset($this->IndexableAuthors)){
+			$this->IndexableAuthors = trim($this->IndexableAuthors ?? '');
```

Comment:
`?? ''` should not be required here because we just checked the same using `isset()` above. If you remove that does phpstan complain?

Reply:
Done. I fixed `IndexableText` above, too. No PHPStan errors.